From 103e504b1c26bafc6a46abdecd846c55e6dceffa Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 27 May 2019 13:46:09 +0700
Subject: [PATCH 163/215] arm64: dts: renesas: r8a774a1: Add the emergency node
 in thermal-zones

This patch enables emerency shutdown support for r8a774a1 SoC.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
index 199f625..8d6e2b2 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
@@ -1599,6 +1599,18 @@
 	};
 
 	thermal-zones {
+		emergency {
+			polling-delay = <1000>;
+			on-temperature = <110000>;
+			off-temperature = <95000>;
+			target_cpus =	<&a57_1>,
+					<&a53_0>,
+					<&a53_1>,
+					<&a53_2>,
+					<&a53_3>;
+			status = "disabled";
+		};
+
 		sensor_thermal1: sensor-thermal1 {
 			polling-delay-passive = <250>;
 			polling-delay = <0>;
-- 
2.7.4

